api/nakadi-event-bus-api.yaml

```diff
@@ -1648,6 +1698,25 @@ definitions:
         description: |
           Offset of the event being pointed to.
 
+  ShiftedCursor:
```

Comment:
Yeah, I would say `offset` doesn't work here, as the object would have two `offset` fields with different meaning (one opaque string, one a numeric difference).